hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/LabelRequest.java

```diff
@@ -0,0 +1,8 @@
+package org.hypertrace.graphql.entity.request;
+
+import org.hypertrace.core.graphql.common.request.AttributeRequest;
+
+public interface LabelRequest {
+
+  AttributeRequest attributeRequest();
```

Comment:
naming, to align with assumptions made later:
```suggestion
  AttributeRequest labelIdArrayAttributeRequest();
```